<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ranking</class>
 <widget class="QWidget" name="Ranking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/resource/picture/背景.jpg);
                         background-repeat: no-repeat;
                         background-position: center;
</string>
  </property>
  <widget class="QComboBox" name="chooseMap">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0 , 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>关卡选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="view">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0 , 0, 0);
background: rgba(255, 255, 255, 50); /* 半透明白色背景 */
            padding: 2px 4px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="kind">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>模式选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="timeBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scoreBtn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>分数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
